docker/tcpv4tracer.py

```diff
@@ -0,0 +1,291 @@
+#!/usr/bin/python
+#
+# tcpv4tracer   Trace TCP IPv4 connections.
+#               For Linux, uses BCC, eBPF. Embedded C.
+#
```

Comment:
add a reference of similar tools on the bcc repo: https://github.com/iovisor/bcc/pull/762